mp/src/game/client/momentum/ui/gameui/BaseMenuPanel.cpp

```diff
@@ -134,6 +139,8 @@ void CBaseMenuPanel::OnGameUIActivated()
 
 void CBaseMenuPanel::OnGameUIHidden()
 {
+    SetPostProcessParams(&m_PostProcessParameters, false);
```

Comment:
Wouldn't this be messing up an existing map's post process, especially if it's using vignette?

Reply:
The map's postprocessing parameters are set on every frame, so I simply block that when the menu is visible and unblock when it's closed.